hw/ip/otbn/doc/_index.md

```diff
@@ -793,7 +814,7 @@ The high-level sequence by which the host processor should use OTBN is as follow
 
 1. Write the OTBN application binary to {{< regref "IMEM" >}}, starting at address 0.
 2. Optional: Write constants and input arguments, as mandated by the calling convention of the loaded application, to {{< regref "DMEM" >}}.
-3. Start the operation on OTBN by writing `1` to {{< regref "CMD.start" >}}.
+3. Start the operation on OTBN by writing `EXECUTE` command to {{< regref "CMD" >}}.
```

Comment:
Nit: Above, you talk about "issuing the SEC_WIPE_IMEM command". Here, it's "write EXECUTE to CMD". Equally happy with either form, but maybe they should be consistent?

Reply:
Good point, I've updated the wording to use "issue xxx command" consistently.